bench/lib/benchmark.js

```diff
@@ -11,8 +12,20 @@ export type Measurement = {
     time: number
 };
 
+type Location = {
+    description: string,
+    tileID: Array<OverscaledTileID>,
+    zoom: number,
+    center: LngLatLike
+};
+
 class Benchmark {
-    constructor() {
+    styleURL: string;
+    locations: ?Array<Location>;
+
+    constructor(styleURL: string, locations: ?Array<Location>) {
```

Comment:
Putting this in the base class feels a little over-generalized to me -- not all benchmarks have a style URL. I'd rather have concrete subclasses initialize the properties they use. There'll be a bit of repetition, but it's more explicit and flexible.